openpype/hosts/maya/plugins/create/create_animation.py

```diff
@@ -17,44 +17,65 @@ class CreateAnimation(plugin.Creator):
     label = "Animation"
     family = "animation"
     icon = "male"
-    write_color_sets = False
-    write_face_sets = False
-    include_parent_hierarchy = False
-    include_user_defined_attributes = False
+    includeUserDefinedAttributes = False
+    eulerFilter = True
+    noNormals = False
+    preRoll = False
+    renderableOnly = False
+    uvWrite = True
+    writeColorSets = False
+    writeFaceSets = False
+    wholeFrameGeo = False
+    worldSpace = True
+    writeVisibility = True
+    writeUVSets = True
+    writeCreases = False
+    dataFormat = "ogawa"
+    step = 1.0
+    attr = ""
+    attrPrefix = ""
+    stripNamespaces = True
+    verbose = False
+    preRollStartFrame = 0
+    farm = False
+    priority = 50
+    includeParentHierarchy = False  # Include parent groups
+    refresh = False  # Default to suspend refresh.
+    visibleOnly = False  # only nodes that are visible
```

Comment:
With exposing this insane amount of attributes to the artist I'd actually want to argue we will be needing studio settings as well which can enable/disable a specific attribute to be visible to the artist to change at all. I'd go crazy debugging whatever the artists start toggling across all these parameters in workfiles.

How can/should I start avoid exposing this to artists? :)

Also, `dataFormat` should definitely be an enum. Right?

Reply:
This might be more trivial with thew new publisher since there you can have the attribute, but also hide it from exposing to the artist but still provide a default value I think.

Reply:
Totally, but still cant wait for the merge.